src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractCohortEngine.java

```diff
@@ -46,8 +46,8 @@
     private final Long maxLocation;
     private final TableReference filteringTableRef;
     private final ReferenceDataSource refSource;
-    private double vqsLodSNPThreshold = 0;
-    private double vqsLodINDELThreshold = 0;
+    private Double vqsLodSNPThreshold;
+    private Double vqsLodINDELThreshold;
```

Comment:
with this change to a nullable type... do you have to do any safety checks for nulls?

Reply:
thanks!